cmd/admin-rpc-client.go

```diff
@@ -39,6 +40,7 @@ type adminCmdRunner interface {
 	Restart() error
 	ListLocks(bucket, prefix string, duration time.Duration) ([]VolumeLockInfo, error)
 	ReInitDisks() error
+	Uptime() (time.Time, error)
```

Comment:
@vadmeste Makes sense, I shall update the PR to use ``time.Duration`` everywhere.